pyproject.toml

```diff
@@ -48,8 +48,8 @@ colors = ["colorama"]
 plugins = ["setuptools"]
 
 [tool.poetry.dev-dependencies]
-bandit = ">=1.6"
-black = ">=22.6.0"
+bandit = ">=1.7.7"
+black = "24.3.0"
```

Comment:
Did you intend to lock this to an specific, old version of black? Could it be `>=24.3.0,<25`?

Reply:
> Did you intend to lock this to an specific, old version of black? Could it be `>=24.3.0,<25`?

another mistake, also fixed with 5760cc9, I didn't both fixing a highes version as there wasn't one before